### PR TITLE
RBN-75 : Finish cleanup

### PR DESCRIPTION
**Pull request by maxencije**
### What did you implement?

- Implemented folder structure for further development
- Deleted unnecessary files and code blocks

### More info

- Jira: [RBN-75](https://productdock.atlassian.net/jira/software/projects/RBN/boards/11?selectedIssue=RBN-75)

- Figma: [Design]()

### CODE REVIEW

**Review by Aleksandra-Romanic — DISMISSED** [review]

**Review by ssukovic — COMMENTED** [review]
Please capitalise folder names which contain components (Footer and Homepage folder). It's not mandatory, just usual convention for React projects

**Review by alestojanovic — COMMENTED** [review]
Github is a little buggy when it comes to changes as capitalising letters without any changes to the word itself. I assume that changes are applied locally but to apply changes to github repo you will need to change the name of folder then change it back to correct form. Example footer -> futer -> Footer as github will not register a change like footer -> Footer.

**Review by ssukovic — DISMISSED** [review]
🚀

File: src/App.tsx [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <Routes>
-          <Route path="/" element={<Homepage />} />
-        </Routes>
-      </header>
-    </div>
+    <header className="App-header">
```
Comment (ssukovic) [inline src/App.tsx]: Remove this header too

**Review by ssukovic — DISMISSED** [review]

**Review by isidora-b — DISMISSED** [review]

**Review by dunjagojkovic — APPROVED** [review]

**Review by isidora-b — APPROVED** [review]

**Pull request merged**